src/structures/api/Match.ts

```diff
@@ -113,13 +113,33 @@ export class Match {
     this.region = this._regionFromPlatformId(data.info.platformId);
     this.queue = client.queues.find((q) => q.queueId === data.info.queueId)!;
     this.tournamentCode = data.info.tournamentCode;
+
     const blueTeamData = data.info.teams.find((t) => t.teamId === 100)!;
-    const redTeamData = data.info.teams.find((t) => t.teamId === 200)!;
-    const blueTeamParticipants = data.info.participants.filter((p) => p.teamId === 100);
-    const redTeamParticipants = data.info.participants.filter((p) => p.teamId === 200);
-    this.teams = new Collection<'blue' | 'red', Team>();
-    this.teams.set('blue', new Team(blueTeamData, blueTeamParticipants, champions, items, runeTrees, summonerSpells));
-    this.teams.set('red', new Team(redTeamData, redTeamParticipants, champions, items, runeTrees, summonerSpells));
+
+    // as per #70, it is not possible to check for the ARENA map the efficient way, so it is hardcoded
+    if (data.info.mapId === 30) {
+      const teamData = blueTeamData;
```

Comment:
Note: Every teams data is stored in the blueTeam for arena - the data is overlapping, but there is API wise no way to separate e.g. bans anyways

Reply:
It is what it is, can only make the lib have what is given from the API.

Reply:
Just make sure that the docs point out this clarification.